exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/parser/UnsupportedOperatorsVisitor.java

```diff
@@ -305,6 +313,105 @@ public SqlNode visit(SqlCall sqlCall) {
     return sqlCall.getOperator().acceptCall(this, sqlCall);
   }
 
+  private void checkRollupCubeGrpSets(SqlSelect sqlSelect) {
+    final ExprFinder rollupCubeGrpSetsFinder = new ExprFinder(RollupCubeGrpSets);
+    sqlSelect.accept(rollupCubeGrpSetsFinder);
+    if (rollupCubeGrpSetsFinder.find()) {
+      unsupportedOperatorCollector.setException(SqlUnsupportedException.ExceptionType.FUNCTION,
```

Comment:
I know that other Unsupported errors have pointed to JIRAs that are marked 'resolved fixed' because they were about throwing such an exception.  Ideally, though, we should create an enhancement JIRA (such as 'Add support for ROLLUP') and point to that in the error message.